cms/static/sass/views/_container.scss

```diff
@@ -331,3 +331,156 @@
+    background: #F2F8FB;
+    overflow: hidden;
+
+    button:not(.button-backward) {
+      padding: 0;
+      border: none;
+    }
+
+    .button-backward {
+      @include float(left);
+      padding-top: 0;
+      border-top: none;
+
+      &:disabled {
+        color: #d9d9d9;
+      }
+
+      .back-fa-icon {
+        @include rtl {
+          @include transform(rotate(180deg));
+        }
+      }
+    }
+
+    .breadcrumbs {
+      @include float(left);
+      @include margin-left($baseline/4);
+      @include padding-left($baseline/2);
+      @include border-left(1px solid #D9D9D9);
+
+      .bc-container {
+        .breadcrumb-fa-icon {
+          width: ($baseline);
+          @include rtl {
+            text-align: left;
+            &.last {
+                text-align: right;
+            }
+          }
+        }
+
+        .breadcrumb-fa-icon:not(.last) {
+          @include rtl {
+            @include transform(rotate(180deg));
+          }
+        }
+        // setting parent font-size to 0 and then setting child font-size is needed to remove the
+        // visible space between inline elements(span.fa-angle-right and button.parent-nav-button)
+        // https://davidwalsh.name/remove-whitespace-inline-block
+        font-size: 0;
+        & > * {
+          @include font-size(14);
+        }
+
+        &.last {
+          .parent-displayname {
+            @include font-size(18);
+          }
+        }
+      }
+
+      @for $i from 2 to 5 {
+        .bc-#{$i} {
+          @include padding-left($baseline * ($i - 1));
+        }
+      }
+
+      .bc-container:not(.last) {
+        button, .parent-displayname {
+          text-decoration: underline;
+          color: $ui-link-color;
+        }
+      }
+    }
+  }
+
+  .category-text {
+    @include font-size(14);
+    color: #414141;
+    @include margin-left($baseline/2);
+  }
+
+  .xblock-items-container {
+    max-height: ($baseline*15);
+    overflow-y: auto;
+
+    .xblock-item {
+      & > * {
+        width: 100%;
+        color: #065683;
+      }
+
+      .component {
+        display: block;
+        color: #414141;
+      }
+
+      .button-forward, .component {
+        border: none;
+        padding: ($baseline/2);
+      }
+
+      .button-forward {
+        .xblock-displayname {
+          @include float(left);
+        }
+
+        .forward-sr-icon {
+          @include float(right);
+
+          @include rtl {
+            @include transform(rotate(180deg));
+          }
+        }
+
+        &:hover, &:focus {
+          background: rgb(242, 248, 251);
+        }
+      }
+    }
+
+    .xblock-no-child-message {
+      display: block;
+      @include text-align(center);
+      padding: ($baseline*2);
+    }
+  }
+
+  .truncate {
+    max-width: 90%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .current-location {
+    @include float(left);
+    @include margin-left($baseline);
+  }
+}
```

Comment:
@muhammad-ammar the UX team reviews the UX from the sandbox, but they don't write or review Sass code. There is nobody on that team who works with Sass, so the engineering team should review Sass changes. If you have specific questions about whether the styling is correct, it is best to ask those in terms of screenshots and/or links to the sandbox.